R/Organism.R

```diff
@@ -796,6 +796,8 @@ setMethod("growth_par", "Bac", function(object, population, j, fbasol, tstep){
 #' @param object An object of class Bac.
 #' @param population An object of class Arena.
 #' @param j The number of the iteration of interest.
+#' @param chemo The vector that contains the prefered substrate.
```

Comment:
It is important to add this parameter, so that the chemotaxis vector accepts more than one compounds without returning error.